components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -410,9 +410,15 @@ SUBROUTINE p3_main_pre_main_loop(kts, kte, kbot, ktop, kdir, log_predictNc, dt,
        dum        = 1.496e-6_rtype * bfb_pow(t(k), 1.5_rtype) / (t(k)+120._rtype)  ! this is mu
        acn(k)     = g*rhow/(18._rtype*dum)  ! 'a' parameter for droplet fallspeed (Stokes' law)
 
-       !specify cloud droplet number (for 1-moment version)
+       !Droplet activation OR specify cloud droplet number (for 1-moment version)
+!+++ JShpund: A condition for minimal cloud drop mass here seems less applicable,
+!             as embrio-clouds forms ontop of CCNs without any prior mass.
+!             Double check with PMC if there is any other reason to enforce that here
+!             as MG2 does (*Erase When Done*)
```

Comment:
@JS-WRF-SBM - I don't understand your question here. I don't see any constraint on cloud drop mass here or in micro_mg2_0.F90. Could you clarify?

Reply:
@PeterCaldwell 
I was pointing out for the inconsistancy between the "`droplet nucleation`" routine and the independent lines in the "`p3_main_pre`_main_loop" routine.